webpack/private/webpack_bundle.bzl

```diff
@@ -421,3 +423,36 @@ def webpack_bundle(
         allow_execroot_entry_point_with_no_copy_data_to_bin = allow_execroot_entry_point_with_no_copy_data_to_bin,
         **kwargs
     )
+
+def linked_webpack(name, node_modules):
```

Comment:
How about `webpack_binary`? I find it better to keep the same prefix for all public api rules from a rule set. `webpack_*` in this case.